src/plugins/intel_gna/tests/functional/shared_tests_instances/skip_tests_check.hpp

```diff
@@ -26,9 +29,20 @@ class GnaLayerTestCheck {
         return lastMsg;
     }
 
-    bool gnaLibVersionLessThan(float verToCmp) {
-        if (gnaLibVer && gnaLibVer < verToCmp) {
-            lastMsg = "GNA library version is less than " + std::to_string(verToCmp);
+    bool gnaLibVersionLessThan(std::string verToCmp) {
+        int verToCmpMajor;
+        int verToCmpMinor;
+
+        if (!verRead) {
+            IE_THROW() << "GnaLayerTestCheck requires initialization with SetUp()";
+        }
+
+        if (sscanf(verToCmp.c_str(), "%d.%d", &verToCmpMajor, &verToCmpMinor) != 2) {
```

Comment:
Don't we need to throw exception here? or an error message at least. It is not clear why we return false if version string has been parsed incorrectly.

Reply:
oh this is test, never mind then